src/core/gps/qgsnmeaconnection.cpp

```diff
@@ -279,6 +279,52 @@ void QgsNmeaConnection::processRmcSentence( const char *data, int len )
       QgsDebugMsgLevel( QStringLiteral( "local time:" ), 2 );
       QgsDebugMsgLevel( mLastGPSInformation.utcDateTime.toLocalTime().toString(), 2 );
     }
+
+    // convert mode to signal (aka quality) indicator
+    // (see https://gitlab.com/fhuberts/nmealib/-/blob/master/src/info.c#L27)
+    if ( result.status == 'A' )
+    {
+      if ( result.mode == 'A' )
+      {
+        mLastGPSInformation.quality = 1;
```

Comment:
any chance we can replace these raw ints with an enum?

Reply:
@nyalldawson , done.